assets/javascripts/discourse/widgets/discourse-reactions-counter.js.es6

```diff
@@ -5,6 +5,14 @@ import { createWidget } from "discourse/widgets/widget";
 import { cancel, later, schedule } from "@ember/runloop";
 import CustomReaction from "../models/discourse-reactions-custom-reaction";
 import MessageBus from "message-bus-client";
+import { addWidgetCleanCallback } from "discourse/components/mount-widget";
+
+let subscriptions = [];
+
+addWidgetCleanCallback("post-stream", () => {
+  subscriptions.forEach((channel) => MessageBus.unsubscribe(channel));
```

Comment:
Why can't we just call `this.unsubscribe` here?

https://github.com/discourse/discourse-reactions/blob/7de5af9e4bef017e4f55cfa346d9485002fe29bf/assets/javascripts/discourse/widgets/discourse-reactions-counter.js.es6#L30-L35

Reply:
sorry replied in chat but this is a global api so it has no access to particular instances